README.md

```diff
@@ -41,3 +41,16 @@ Examples to Learn Python Django
 
 
 Source : <a href="https://www.djangoproject.com/">https://www.djangoproject.com</a>
+
+A very basic file structure in Django:
+--------------------------------------
+└── new_project
+    ├── new_project
+    │   ├── __init__.py
+    │   ├── settings.py
+    │   ├── urls.py
+    │   └── wsgi.py
+    └── manage.py
```

Comment:
Hey @satyam15mishra ,

Add the structure in a code block so that the format remains. Do check the preview as you make the change.

Thanks for contributing to #10 
Feel free to add more examples and docs.